src/localStorage.js

```diff
@@ -0,0 +1,19 @@
+import { tasks } from './taskClass.js';
+// eslint-disable-next-line import/no-cycle
+import addTasks from './showTasks.js';
+
+const saveTasks = () => {
+  localStorage.setItem('tasksKey', JSON.stringify(tasks));
+};
+
+const getTasks = () => {
+  if (localStorage.getItem('tasksKey')) {
+    const loadTasks = JSON.parse(localStorage.getItem('tasksKey'));
+    loadTasks.forEach((t) => {
+      tasks.push(t);
+    });
+    addTasks();
+  }
+};
```

Comment:
- [ ] Please add the added taskskey you have the localStorage but it's not displayed in the Showtask.